.yarn/versions/bb0efa48.yml

```diff
@@ -0,0 +1,5 @@
+releases:
+  "@radix-ui/react-tooltip": patch
```

Comment:
Given that we will be adjusting these in the release I just marked as patch, is that correct?